operators/src/util/raster_stream_to_geotiff.rs

```diff
@@ -285,7 +285,7 @@ impl<P: Pixel + GdalType> GdalDatasetHolder<P> {
             full_path: placeholder_path,
             placeholder: placeholder.to_string(),
             time_placeholder: GdalSourceTimePlaceholder {
-                format: DateTimeParseFormat::custom("%Y-%m-%dT%H-%M-%S".to_string()),
+                format: DateTimeParseFormat::custom("%Y-%m-%dT%H-%M-%S-%3f".to_string()),
```

Comment:
Could you add a test, where it is checked (in a temp folder) that files with specific (expected) names exist?

It is to some degree concerning that the file names changed but there was no test to adapt.

Reply:
ef3749e2769b7bf09ce3e11dbbeb3e597c4e97e5